services/intersection-api/jpo-conflictvisualizer-api/.gitignore

```diff
@@ -0,0 +1,38 @@
+HELP.md
+target/
+bin/
+!.mvn/wrapper/maven-wrapper.jar
```

Comment:
chore: it looks like we also need to track `.mvn/wrapper/maven-wrapper.properties` otherwise we'll get the following error 
```
./mvnw: 223: cannot open /home/orion/repos/CDOT-CV/jpo-cvmanager/services/intersection-api/jpo-conflictvisualizer-api/.mvn/wrapper/maven-wrapper.properties: No such file
/home/orion/repos/CDOT-CV/jpo-cvmanager/services/intersection-api/jpo-conflictvisualizer-api/.mvn/wrapper/maven-wrapper.jar: No such file or directory
Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
```

Reply:
.mvn/wrapper required files are now tracked